src/dmn-viewer/index.ts

```diff
@@ -0,0 +1,59 @@
+import { css, LitElement, unsafeCSS } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from "../shared/styles/reset";
+
+import Viewer from 'dmn-js';
+import ViewerSharedCss from 'dmn-js/dist/assets/dmn-js-shared.css';
+import ViewerDrdCss from 'dmn-js/dist/assets/dmn-js-drd.css';
+import ViewerDecisionTableCss from 'dmn-js/dist/assets/dmn-js-decision-table.css';
+import ViewerLiteralExpressionCss from 'dmn-js/dist/assets/dmn-js-literal-expression.css';
+import ViewerDMNCss from 'dmn-js/dist/assets/dmn-font/css/dmn.css';
+
+@customElement('dmn-viewer')
+export class DMNViewer extends LitElement {
+  private _viewer!: any;
+
+  @property({ attribute: "data-xml" })
+  xml!: string
+
+  constructor() {
+    super();
+  }
+
```

Comment:
```suggestion
```

Reply:
Check this one, it says outdated, cannot commit suggestion.

Reply:
it looks like this file has been deleted in this commit https://github.com/biz-dev-ops/web-components/pull/96/commits/6ddb9282180e8aa7c419a49d8339ec00baa2e445. Not sure if it was intentional.

Reply:
My bad. Will recover the files this weekend.

Reply:
Files recovered.